lib/google/longrunning/operations_client.rb

```diff
@@ -114,7 +114,8 @@ def initialize \
           client_config: {},
           timeout: DEFAULT_TIMEOUT,
           lib_name: nil,
-          lib_version: ""
+          lib_version: "",
+          _headers: {}
```

Comment:
I think `metadata` makes more sense than `_headers`.

```suggestion
          metadata: {}
```

Reply:
I can understand the naming. It's specifically injecting http headers. Maybe `extra_headers`?

Reply:
I think of it as the inverse. Users can provide common headers/metadata to use on every RPC call when creating the GAPIC client, and the GAPIC client adds the `x-goog-api-client` header to the headers/metadata that users provide.

Additionally, users can provide metadata on the individual RPC calls, and the common medatadata set on the client is merged with the per-RPC metadata.